Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/MicrosoftGraphFiles.py

```diff
@@ -815,35 +611,36 @@ def delete_file_command(client, args):
 
 
 def main():
-    # Should we use SSL
-    verify_certificate = not demisto.params().get("insecure", False)
-    proxy = demisto.params().get("proxy", False)
-    base_url = demisto.params().get("host", False)
-    LOG(f"Command being called is {demisto.command()}")
+    params: dict = demisto.params()
+    base_url: str = params.get('host', '').rstrip('/') + '/v1.0/'
+    tenant = params.get('tenant_id')
+    auth_id = params.get('auth_id')
+    enc_key = params.get('enc_key')
+    use_ssl: bool = not params.get('insecure', False)
+    proxy: bool = params.get('proxy', False)
+    self_deployed: bool = params.get('self_deployed', False)
+    ok_codes: tuple = (200, 204, 201)
+
     try:
-        # client = Client(BASE_URL, proxy=proxy, verify=verify_certificate)
-        client = Client(
-            base_url=base_url + "/v1.0",
-            verify=verify_certificate,
-            proxy=proxy,
-            ok_codes=(200, 204, 201),
-        )
+        client = MsGraphClient(base_url=base_url, tenant_id=tenant, auth_id=auth_id, enc_key=enc_key, app_name=APP_NAME,
+                               verify=use_ssl, proxy=proxy, self_deployed=self_deployed, ok_codes=ok_codes)
+
+        LOG(f"Command being called is {demisto.command()}")
+
         if demisto.command() == "test-module":
             # This is the call made when pressing the integration Test button.
             result = module_test(client)
             demisto.results(result)
-
         elif demisto.command() == "msgraph-delete-file":
             readable_output, raw_response = delete_file_command(client, demisto.args())
```

Comment:
`raw_response` is not used here

Reply:
should I use:
` readable_output, _ = delete_file_command(client, demisto.args())`
or change the return value of that command?

Reply:
change the line below to `return_outputs(readable_output=readable_output, raw_response=raw_response)`